services/src/main/java/org/keycloak/protocol/oidc/endpoints/UserInfoEndpoint.java

```diff
@@ -132,7 +132,7 @@ private Response issueUserInfo(String tokenString) {
         ClientModel clientModel = realm.getClientByClientId(token.getIssuedFor());
         UserModel userModel = userSession.getUser();
         AccessToken userInfo = new AccessToken();
-        tokenManager.transformAccessToken(session, userInfo, realm, clientModel, userModel, userSession, clientSession);
+        tokenManager.transformUserInfoAccessToken(session, userInfo, realm, clientModel, userModel, userSession, clientSession);
```

Comment:
`transformUserInfoAccessToken` -> does the userinfo really represent an AccessToken? technically yes but actually  are different things...